### PR TITLE
Added support for Captcha v2

### PR DESCRIPTION
**Pull request by wizzard**

### CODE REVIEW

File: src/mod_perimeterx.c [hunk 1]
```diff
@@ -918,12 +916,11 @@ static void set_app_id_helper(apr_pool_t *pool, px_config *conf, const char *app
     conf->app_id = app_id;
     conf->base_url = apr_psprintf(pool, DEFAULT_BASE_URL, app_id, NULL);
     conf->risk_api_url = apr_pstrcat(pool, conf->base_url, RISK_API, NULL);
-    conf->captcha_api_url = apr_pstrcat(pool, conf->base_url, CAPTCHA_API, NULL);
     conf->activities_api_url = apr_pstrcat(pool, conf->base_url, ACTIVITIES_API, NULL);
     const char *reverse_prefix =  &app_id[2];
     conf->xhr_path_prefix = apr_psprintf(pool, "/%s/xhr", reverse_prefix);
     conf->client_path_prefix = apr_psprintf(pool, "/%s/init.js", reverse_prefix);
-    conf->client_exteral_path = apr_psprintf(pool, "//client.perimeterx.net/%s/main.min.js", app_id);
+    conf->client_external_path = apr_psprintf(pool, "//client.perimeterx.net/%s/main.min.js", app_id);
```
Comment (wizzard) [inline src/mod_perimeterx.c]: is this correct that for HTTP clients' requests will we will send internal HTTP requests to client.perimeterx.net ? shouldn't we use HTTPS ?

File: src/px_template.c [hunk 2]
```diff
@@ -10,12 +10,7 @@
...
...ou are a human</h1> </div> </div> <div class=\"content-wrapper\"> <div class=\"content\"> <div id=\"px-captcha\"> </div> <p> Access to this page has been denied because we believe you are using automation tools to browse the website. </p> <p> This may happen as a result of the following: </p> <ul> <li> Javascript is disabled or blocked by an extension (ad blockers for example) </li> <li> Your browser does not support cookies </li> </ul> <p> Please make sure that Javascript and cookies are enabled on your browser and that you are not blocking them from loading. </p> <p> Reference ID: #{{refId}} </p> </div> </div> <div class=\"page-footer-wrapper\"> <div class=\"page-footer\"> <p> Powered by <a href=\"https://www.perimeterx.com/whywasiblocked\">PerimeterX</a> , Inc. </p> </div> </div> </section> <!-- Px --> <script> window._pxAppId = '{{appId}}'; window._pxJsClientSrc = '{{{jsClientSrc}}}'; window._pxFirstPartyEnabled = {{firstPartyEnabled}}; window._pxVid = '{{vid}}'; window._pxUuid = '{{uuid}}'; window._pxHostUrl = '{{{hostUrl}}}'; </script> <script src=\"{{{blockScript}}}\"></script> <!-- Custom Script --> {{#jsRef}} <script src=\"{{jsRef}}\"></script> {{/jsRef}} </body> </html>";
```
Comment (wizzard) [inline src/px_template.c]: found some differences with PHP code. Please verify that this is the latest js template version.

File: src/px_template.c [hunk 3]
```diff
@@ -151,55 +169,14 @@ int render_template(const char *tpl, char **html, const request_context *ctx, co
         .customLogo = conf->custom_logo,
         .cssRef = conf->css_ref,
         .jsRef = conf->js_ref,
-        .jsClientSrc = conf->first_party_enabled ? conf->client_path_prefix : conf->client_exteral_path,
-        .firstPartyEnabled = conf->first_party_enabled ? "1" : NULL,
+        .jsClientSrc = conf->first_party_enabled ? conf->client_path_prefix : conf->client_external_path,
+        .firstPartyEnabled = conf->first_party_enabled ? "true" : "false",
         .logoVisibility = conf->custom_logo ? visible : hidden,
         .captchaType = captcha_type_str(conf->captcha_type),
```
Comment (nitzanpx) [inline src/px_template.c]: we can  remove this from the config

File: src/px_json.c [hunk 4]
```diff
@@ -57,8 +54,8 @@ static const char *ACTION_STR[] = {
 };
 
 static const char *CAPTCHA_TYPE_STR[] = {
```
Comment (nitzanpx) [inline src/px_json.c]: I think this is not relevant anymore

**Pull request merged**